### PR TITLE
Add NEWS entry for #29462

### PR DESCRIPTION
**Pull request by simonbyrne**
see #29462

### CODE REVIEW

**Comment by JeffBezanson** [comment]
Oh, is that really the only visible change from that PR? Might not be worth mentioning if so.

**Comment by simonbyrne** [comment]
Effectively, yes. I'll leave it up to you to decide.

File: NEWS.md [hunk 1]
```diff
@@ -106,6 +106,9 @@ Standard library changes
 
 #### Statistics
   * `mean` and `var` now handles the empty case ([#29033]).
+
```
Comment (fredrikekre) [inline NEWS.md]: 2 spaces here

**Comment by simonbyrne** [comment]
please reopen if people think it is worth adding.

**Pull request closed without merging**